recipes-connectivity/mbed-edge-core/files/imx8mmevk/target.cmake

```diff
@@ -2,7 +2,9 @@ MESSAGE ("Building Yocto Linux target")
 SET (OS_BRAND Linux)
 SET (MBED_CLOUD_CLIENT_DEVICE "imx8")
 SET (PAL_TARGET_DEVICE "imx8")
-
+SET (MBED_CLOUD_CLIENT_MIDDLEWARE curl)
+SET (PLATFORM_TARGET Yocto_Generic_YoctoLinux_mbedtls)
+  
```

Comment:
This should be in a if(FOTA_ENABLE) block as it is not required if the update client is used.

Reply:
You mean line 5? The curl line? I wonder if the whole line is needed, would it be enough to have the Yocto_Generic alone? Needs a build trial, which I don't want to do with a laptop.

Reply:
I mean BOTH lines.